deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml

```diff
@@ -78,6 +94,7 @@ spec:
           required:
           - backingServiceSelector
           - applicationSelector
+          - customEnvVar
```

Comment:
Is ```customEnvVar``` intended to be required? In cases where volume mounts are used exclusively, would there be any need for ```customEnvVar```?

Reply:
Nope, this should not be `required`.

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/159/commits/723e0b74a21e75a8208b65cbb544c12cbfcac88a